core/src/test/java/org/hibernate/ogm/backendtck/associations/collection/manytomany/ManyToManyExtraTest.java

```diff
@@ -22,8 +23,8 @@
 /**
  * @author Emmanuel Bernard &lt;emmanuel@hibernate.org&gt;
  */
-@SkipByGridDialect(
-		value = { GridDialectType.CASSANDRA },
+@SkipByDatastoreProvider(
```

Comment:
Is this change (and others of the same kind) actually needed? There is one datastore provider for Cassandra and one dialect, so it shouldn't matter whether it's skipped by dialect or provider? This should only make a difference where we have several dialects (Redis) or providers (Mongo).